std/conv.d

```diff
@@ -207,6 +207,13 @@ template to(T)
     {
         return toImpl!T(arg);
     }
+
+    // Fix issue 16108
+    T to(S)(ref S arg)
+        if (isAggregateType!S && !isCopyable!S)
+    {
+        return toImpl!T(arg);
+    }
 }
```

Comment:
Can these not be merged into a single
```
T to(A...)(auto ref A args)
```
i.e. amending the very first overload and removing the "fix issue" overloads?